server/gitrest/packages/gitrest-base/src/utils/isomorphicgitManager.ts

```diff
@@ -276,7 +276,14 @@ export class IsomorphicGitRepositoryManager implements IRepositoryManager {
             ]);
             return conversions.refToIRef(resolvedRef, expandedRef);
         } catch (err) {
-            winston.error(`getRef error: ${safeStringify(err, undefined, 2)} repo: ${this.repoName} ref: ${refId}`);
+            Lumberjack.error(
+                "getRef error",
+                {
+                    ...this.lumberjackBaseProperties,
+                    repo: this.repoName,
+                    ref: refId,
```

Comment:
It might be redundant, but again, I don't think it hurts to add this here, especially since we are running a lot of validations and tests around GitRest/isomorphic-git. It's probably worth it to have redundant data now than having to come back later add more logs because we missed something. After we are confident with the new solution, we can trim logs and remove the redundant properties.